tests/cases/onboarders_care_platform/case_onboarders_care_platform_account.yaml

```diff
@@ -62,4 +62,95 @@ TestManuallyLogOutOfThePortalOnboarder:
     Id: $PAGE.care_platform_logout_modal.cancel_logout_button$
   # Repeat the steps from above and log out.
   - Type: case
-    Value: CarePlatformLogout
\ No newline at end of file
+    Value: CarePlatformLogout
```

Comment:
Please add a breakline between tests to keep format

Reply:
Done